File: src/main/java/com/biblioteca/proyecto_spring_redits/Model/Cliente.java
package com.biblioteca.proyecto_spring_redits.Model;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.Id;
import lombok.Getter;
import lombok.Setter;

@Entity
@Getter
@Setter
public class Cliente {

    @Id
    @Column(name = "id")
    @GeneratedValue(strategy = jakarta.persistence.GenerationType.IDENTITY)
    private int id;

    @Column(name = "nombre", nullable = false, length = 50)
    private String nombre;

    @Column(name = "apellido", nullable = false, length = 50)
    private String apellido;

    @Column(name = "direccion", nullable = false, length = 200)
    private String direccion;

    @Column(name = "telefono", nullable = false, length = 10, columnDefinition = "CHAR(10)")
    private String telefono;
}

File: src/main/resources/static/Models/Cliente.ts
export class Cliente {
    id: number;
    nombre: string;
    apellido: string;
    direccion: string;
    telefono: string;

    constructor(id: number, nombre: string, apellido: string, direccion: string, telefono: string) {
        this.id = id;
        this.nombre = nombre;
        this.apellido = apellido;
        this.direccion = direccion;
        this.telefono = telefono;
    }
}